<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WdgExportStoryboard</class>
 <widget class="QWidget" name="WdgExportStoryboard">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>430</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Export Storyboard</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Frame number of first item to render:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QSpinBox" name="spinboxFirstItem"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_5">
       <property name="text">
        <string>Frame number of last item to render:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QSpinBox" name="spinboxLastItem"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Rows per page :</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QSpinBox" name="spinboxRow"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Columns per page :</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QSpinBox" name="spinboxColumn">
       <property name="value">
        <number>0</number>
       </property>
      </widget>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="label_6">
       <property name="text">
        <string>Font Size :</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QSpinBox" name="spinboxFontSize"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Page Size :</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QComboBox" name="cmbPageSize">
       <item>
        <property name="text">
         <string>A4</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A2</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A3</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>A5</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="6" column="0">
      <widget class="QLabel" name="label_7">
       <property name="text">
        <string>Page Orientation :</string>
       </property>
      </widget>
     </item>
     <item row="6" column="1">
      <widget class="QComboBox" name="cmbPageOrient">
       <item>
        <property name="text">
         <string>Portrait</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Landscape</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="7" column="0">
      <widget class="QCheckBox" name="chkUseSVGLayout">
       <property name="text">
        <string>Specify Layout Using SVG file </string>
       </property>
      </widget>
     </item>
     <item row="8" column="0">
      <widget class="QLabel" name="label_8">
       <property name="text">
        <string>SVG file for specifying layout :</string>
       </property>
      </widget>
     </item>
     <item row="8" column="1">
      <widget class="KisFileNameRequester" name="svgLayoutFileName" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="9" column="0">
      <widget class="QLabel" name="lblExportFileName">
       <property name="text">
        <string>Export File Name : </string>
       </property>
      </widget>
     </item>
     <item row="9" column="1">
      <widget class="KisFileNameRequester" name="exportFileName" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
      </widget>
     </item>
     <item row="10" column="0">
      <widget class="QLabel" name="lblSvgFileBaseName">
       <property name="text">
        <string>SVG file base name : </string>
       </property>
      </widget>
     </item>
     <item row="10" column="1">
      <widget class="QLineEdit" name="svgFileBaseName"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>KisFileNameRequester</class>
   <extends>QWidget</extends>
   <header>kis_file_name_requester.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
